great_expectations_cloud/agent/agent.py

```diff
@@ -343,6 +345,10 @@ def _handle_event_as_thread_exit(
         event_context.processed_successfully()
         self._current_task = None
 
+    def _get_processed_by(self) -> str:
+        """Return the name of the service that processed the event."""
+        return "runner" if self._config.queue == "gx-runner" else "agent"
+
```

Comment:
This seemed like the easiest way to accomplish adding processed_by here. I'm open to other ideas, though, especially if we don't want the agent to know about the runner queue.